app/models/court_type.rb

```diff
@@ -9,9 +9,9 @@ class CourtType < ActiveRecord::Base
 
   validates :value, presence: true
 
-  value :constitutional, 'Ústavný'
-  value :supreme,        'Najvyšší'
-  value :specialized,    'Špecializovaný'
-  value :regional,       'Krajský'
-  value :district,       'Okresný'
+  value :constitutional, I18n.t('.constitutional')
```

Comment:
path should be something like `court.consitutional` or `court_type.constitutional` and apply that for other types as well.